### PR TITLE
Fix NO_PROXY flake

### PR DESCRIPTION
**Pull request by jhadvig**
Have seen the e2e test for the proxy flaking lately with following error:
```
unable to find environment variable: wanted NO_PROXY
```

Need to do some investigation, so marking as WIP

/assign @benjaminapetersen

### CODE REVIEW

**Comment by benjaminapetersen** [comment]
Great, I see it as well [here](https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_console-operator/303/pull-ci-openshift-console-operator-master-e2e-aws-operator/1241) for #303.

File: test/e2e/proxy_test.go [hunk 1]
```diff
@@ -76,10 +75,25 @@ func TestProxy(t *testing.T) {
 		return false, nil
 	})
 
-	for _, err = range framework.CheckEnvVars(proxyEnvVars["clusterVars"], newConsoleDeployment.Spec.Template.Spec.Containers[0].Env, true) {
-		t.Fatal(err)
-	}
-
+	counter := 0
+	maxCount := 60
```
Comment (benjaminapetersen) [inline test/e2e/proxy_test.go]: So this simply takes a while to propagate correct?

Comment (benjaminapetersen) [reply test/e2e/proxy_test.go]: Ready to remove the `[WIP]`?

**Comment by jhadvig** [comment]
Ran the test couple of time to see if the flake will show up. Since it didn't occur I think It's ready for review.
@benjaminapetersen PTAL

File: test/e2e/proxy_test.go (hunk 1 above)
Comment (jhadvig) [reply test/e2e/proxy_test.go]: > So this simply takes a while to propagate correct?
yes

**Pull request merged**